src/ontology/mondo-edit.obo

```diff
@@ -372925,6 +373285,7 @@ xref: SCTID:722051004 {source="MONDO:equivalentTo"}
 xref: UMLS:CN227640 {source="MONDO:equivalentTo"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-5002-8648"} ! syndromic disease
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6651" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6878" xsd:anyURI
```

Comment:
In this first commit (https://github.com/monarch-initiative/mondo/pull/6887/commits/2ec777ab79bcd005fb6de86ce9502e1223b893da), I only see the ticket being added, but no new `is_a`.

Reply:
yes, I think it is because the annotation was already in the ontology. It is in line 372286 (syndromic disease, from Katie's orcid)

Reply:
oh, ok. in this first commit i look for pairs of ticket and new `is_a` being added.